cfme/utils/bz.py

```diff
@@ -207,6 +209,58 @@ def resolve_blocker(self, blocker, version=None, ignore_bugs=None, force_block_s
                 return bug
         return None
 
+    def set_flags(self, idlist, flags):
+        bz_creds = credentials.get("bugzilla", None)
+
+        # login to bzapi
+        if not bz_creds:
```

Comment:
Optional: I would not raise error just with msg exit.

Reply:
See my comment about this above

Reply:
Not sure what comment above you're talking about - can you put an in-line comment here as to why we're raising and not just logging.

Reply:
I think this is still applicable here

Reply:
Sorry, missed that message @mshriver. We're raising the error so that the user will know why their command has failed if there are no bugzilla credentials available